surf.deckr.deckr.yaml

```diff
@@ -0,0 +1,203 @@
+app-id: surf.deckr.deckr
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+command: deckr
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+
+modules:
+  - name: deckr
+    buildsystem: simple
+    build-commands:
+      # - mkdir -p deckr
+      # - tar -xf Deckr-Linux-Portable.tar.gz -C deckr
+      - mv deckr /app/deckr
+      - echo $(ls /app/deckr)
+
+      - chmod +x /app/deckr/deckr
+      - mkdir -p /app/bin
+      - ln -s /app/deckr/deckr /app/bin/deckr
+
+      - mv gpt /app/gpt
+      - echo $(ls /app/gpt)
+
+      - install -Dm644 surf.deckr.deckr.metainfo.xml -t /app/share/metainfo/
+      - install -Dm644 app_icon_32.png /app/share/icons/hicolor/32x32/apps/surf.deckr.deckr.png
+      - install -Dm644 app_icon_64.png /app/share/icons/hicolor/64x64/apps/surf.deckr.deckr.png
+      - install -Dm644 app_icon_128.png /app/share/icons/hicolor/128x128/apps/surf.deckr.deckr.png
+      - install -Dm644 app_icon_256.png /app/share/icons/hicolor/256x256/apps/surf.deckr.deckr.png
+      - install -Dm644 app_icon_512.png /app/share/icons/hicolor/512x512/apps/surf.deckr.deckr.png
+      - install -Dm644 surf.deckr.deckr.desktop -t /app/share/applications/
+    sources:
+      - type: archive
+        url: https://get.cocoatech.com/dashboard/Deckr-Linux-Portable.tar.gz
```

Comment:
What is the relation of this with deckr.surf ? If this is the main app shouldn't the ID be `com.cocoatech` or `io.cocoatech`?

Reply:
The ID is a blocker, if the main app here is coming from cocoatech.com why does it need a surf.deckr ID?

That links to my S3 account. I could put in a raw S3 download url, but I use this for all my stuff.

I need the surf.decker since I want to verify the app to the apps website. https://deckr.surf/

Reply:
One company two products.  I use that s3 for all my downloads.

Reply:
I need to establish some relationship with the domains here. Do you have control over https://www.cocoatech.io/ ? If so how do I verify that?

> I need the surf.decker since I want to verify the app to the apps website. https://deckr.surf/

Can you please put a txt file or something under https://deckr.surf/.well-known/ so that I know you control that too?

Reply:
https://deckr.surf/.well-known/org.flathub.VerifiedApps.txt

working on cocoatech.io and .com next

Reply:
https://www.cocoatech.io/sandbox

Reply:
See last line of this blog post
https://cocoatech.com/blog/#/

Reply:
Thanks!